tests/integration/controller/suite_test.go

```diff
@@ -81,9 +81,10 @@ var _ = BeforeSuite(func() {
 		if len(parts) < 2 {
 			Fail(fmt.Sprintf("Invalid ISTIO_VERSION env var. Expected version of the form '1.23.0'. Got: %s", istioVersion))
 		}
+		parts2 := strings.Split(parts[1], "-") // For the 1.23-latest format
```

Comment:
Does this code now require that we always pass in `X.Y-latest`? I don't think it does (which is good) but I want to make sure. I don't think we want to require `-latest` always in the tests just in case in the future we do pass in X.Y.Z.

Which brings me to my second point - look at the Fail msg in the line above : `Expected version of the form '1.23.0'` -- is that true anymore? I don't think it is - I think we also allow `1.23-latest` now.

Reply:
Right! You can still use X.Y.Z. I've updated the message.